addon/models/osf-model.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
I can understand treating this as a base class, but it seems that ember data compiles a list of available models in the app based on the contents of this directory. 

![screen shot 2016-05-27 at 3 22 43 pm](https://cloud.githubusercontent.com/assets/2957073/15618967/1bbe59d8-241f-11e6-80e2-8ea047b711c4.png)


I'm not 100% comfortable with the potential consequences of registering a model we don't intend to be used as such- but it's more a sense of ember magic that could come back to haunt us, rather than a concrete blocking concern.